tasks/install-enketo.yml

```diff
@@ -55,22 +55,23 @@
   become: true
   become_user: "{{ enketo_user }}"
   ansible.builtin.replace:
-    path: "{{ enketo_checkout_path }}/app/views/styles/theme-{{ enketo_theme_to_replace }}/_variables.scss"
+    path: "{{ enketo_checkout_path }}/app/views/styles/theme-kobo/_variables.scss"
```

Comment:
Should we support replacing the brand color for other themes too?

Reply:
Related issue for a separate PR: https://github.com/onaio/ansible-enketo/issues/38#issue-1163540644